Laba3.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+#include <stdlib.h>
+// For some reason Visual Studio wants scanf to be changed to scanf_s
+#define scanf scanf_s
+
+int main()
+{
+    int n1, n2, k, * arr1, * arr2, * arrd;
```

Comment:
It's better when variables are more descriptive, even if they are longer, like size1, size2, or arrd -> common_elements